validphys2/src/validphys/covmats.py

```diff
@@ -339,6 +351,41 @@ def dataset_inputs_t0_covmat_from_systematics(
         _list_of_central_values=dataset_inputs_t0_predictions
     )
 
+def dataset_inputs_t0_total_covmat(dataset_inputs_loaded_cd_with_cuts,
+    *,
+    data_input,
+    use_weights_in_covmat=True,
+    norm_threshold=None,
+    dataset_inputs_t0_predictions,
+    output_path,
+    theory_covmat_flag,
+    use_user_uncertainties,
+    use_scalevar_uncertainties):
+    exp_covmat = dataset_inputs_covmat_from_systematics(
+        dataset_inputs_loaded_cd_with_cuts,
+        data_input,
+        use_weights_in_covmat,
+        norm_threshold=norm_threshold,
+        _list_of_central_values=dataset_inputs_t0_predictions
+    )
+    if theory_covmat_flag is True:
+        generic_path = None
+        if use_scalevar_uncertainties is True:
+            if use_user_uncertainties is True:
+                generic_path = pathlib.Path("datacuts_theory_theorycovmatconfig_total_theory_covmat.csv")
+            else:
+                generic_path = pathlib.Path("datacuts_theory_theorycovmatconfig_theory_covmat_custom.csv")
+        else:
+            if use_user_uncertainties is True:
+                generic_path = pathlib.Path("datacuts_theory_theorycovmatconfig_user_covmat.csv")
+            else:
+                generic_path = pathlib.Path("datacuts_theory_theorycovmatconfig_theory_covmat_custom.csv")
+        theorypath = pathlib.Path(str(output_path/"tables"/generic_path.relative_to(generic_path.anchor)))
```

Comment:
I guess output_path is already a path so this (I think, please test!) should do the trick.

```suggestion
       theorypath = output_path / "tables" / generic_path
```